### PR TITLE
CONTENT: Mission Set & Fixes

### PR DESCRIPTION
**Pull request by OcelotWalrus**
# CONTENT

## Summary
This PR adds a new mission set about escorting a young woman from Los Gerudos to the Cocorico Village, with a pretty big dialog, where the player learns about some game lore. It also updates a new preferences: `game data analyzing`, which enables or disable the run of integration tests when launching a game. And finally, it adds a new debug command: `$run$dialog$`, which, runs a selected dialog.

## Testing Method
Teleport you to the `point6990` (_-38, 10) map point using the `$teleport$point$` debug command, and refresh the game using the `K` command, until the mission gets offered to you. You can then test every aspect of the dialog. Then teleport you to the `point2075` (_-53, 48) map point to get the mission completed.

## Save File
Any save file will work.

## Performance Impact
N/A

## Check-List
{{If this PR is in progress, write major actions that need to be done here.}}
- [x] Review my own code
- [x] Fix the workflow checks
- [ ] Update BOW-Progression repo

### CODE REVIEW

**Comment by OcelotWalrus** [comment]
Would you mind giving this PR a review @lumbar527? Thanks.

File: data/dialog.yaml [hunk 1]
```diff
@@ -153,28 +153,32 @@ Gerud Ancient Dungeon Reward:
...
 
 Given Back Dog Mitchell:
   conversation:
     - label 1:
       - print(You finally arrive to the 13 Yasward Drive, Los Gerudos, the village of the Coopers. You knock at the door.)
       - wait(2)
-      - print(A bulter finally opens the great doors, and greets you, before asking who you are and what do you want. You explain to him everything, before showing him the dog. Suddently, the bulter shouts:)
-      - print(“Mrs. Cooper! We've found Mitchell back!“\n)
+      - print(A bulter finally opens the great doors, and greets you, before asking who you are and what do you want. You explain to him everything, before showing him the dog. Suddenly, the bulter shouts:)
+      - print(“- Mrs. Cooper! We've found Mitchell back!“\n)
```
Comment (lumbar527) [inline data/dialog.yaml:172]:
```suggestion
      - print(“- Mrs. Cooper! We've found Mitchell again!“\n)
```
```suggestion
      - print(“- Mrs. Cooper! Mitchell is back!“\n)
```
Not sure which you're going for here.

File: data/dialog.yaml [hunk 2]
```diff
@@ -153,28 +153,32 @@ Gerud Ancient Dungeon Reward:
...
-      - print(“Mitchell! Oh my godness! Finally! you're here! Where did you find him bulter?“, says the chubby lady.)
-      - print(“Actually it's me who've found him, in the grand avenue. I read his collar and then I came here.“, you respond to her.)
-      - print(“What is your name? I'm so grateful, and I couldn't be more.“)
-      - print(“My name is $name, and I'm from the Ayar Town, in the far Forlindon Woods. But I'm pretty much sure you haven't heard of this place before.“)
-      - print(“The Forlindon Woods? That is where I used to live, when I was a little girl! But then my dad bought a ranch there. What can I do for you? Is there anything you'd like to have?“, the lady asks)
-      - print(“Please, I dind't do much.“)
-      - print(“You didn't do much? You've saved my dear love! Please, accept these two ingots.“)
+      - print(A chubby lady suddenly goes down the main stairs, running towards the opened doors.)
+      - print(“- Mitchell! Oh my godness! Finally! you're here! Where did you find him bulter?, says the chubby lady.)
+      - print(- Actually it's me who've found him, in the grand avenue. I read his collar and then I came here., you respond to her.)
```
Comment (lumbar527) [inline data/dialog.yaml:176]:
```suggestion
      - print("- Actually it's me who've found him, in the grand avenue. I read his collar and then I came here," you respond to her.)
```
How do you want to use quotes here?

File: data/dialog.yaml [hunk 3]
```diff
@@ -205,34 +206,83 @@ Wanted Bandit Introduction:
     - label 2:
       - "print(You then decide to approach the sheets. You walk a few feet long and arrive to the wall. One of the sheets' inscription reads: “WANTED -- 'The Dagger' -- Dead or alive; 465 gold units“.)"
       - print(You decide to go to the sheriff and ask him about the job. He first says to you a little more piece of information about it:)
-      - print(“A bandint known as 'The Dagger' has murdered one of the Los Gerudos prince 2 months ago. I arrested him and placed him in the Los Gerudos prison. Two das ago, he found a way to dig a hole in his cell's walls and he escaped the prison. Some citizens said having seen him going near the south-western Los Gerudos at the Gerud Forges.“)
+      - print(“- A bandit known as 'The Dagger' has murdered one of the Los Gerudos prince 2 months ago. I arrested him and placed him in the Los Gerudos prison. Two days ago, he found a way to dig a hole in his cell's walls and he escaped the prison. Some citizens said having seen him going near the south-western Los Gerudos at the Gerud Forges.“)
```
Comment (lumbar527) [inline data/dialog.yaml:209]:
```suggestion
      - print(“- A bandit known as 'The Dagger' has murdered one of the Los Gerudos princes 2 months ago. I arrested him and placed him in the Los Gerudos prison. Two days ago, he found a way to dig a hole in his cell's walls and he escaped the prison. Some citizens said having seen him going near the south-western Los Gerudos at the Gerud Forges.“)
```

**Review by lumbar527 — APPROVED** [review]
Seems nice!
(Haven't tested yet though)

File: data/dialog.yaml (hunk 2 above)
Comment (OcelotWalrus) [reply data/dialog.yaml:176]: The quotes start at the beginning of the conversation and finish at the end, so the quotes are only located at the first `print()` method of the conversation and the final one. I'm talking about the characters conversation here.

**Pull request merged**